index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
```

Comment:
- [ ] Please I highly recommend sticking to the required [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=48-838&t=K8YmgDZrsKmtImKI-0). 😄 

_**REQUIRED FIGAM DESIGN**_

![Screenshot (307)](https://github.com/AndresEspi/portafolio/assets/78104544/ec976680-4a87-4b15-85b8-43cd6c6026bf)


_**YOUR DESIGN**_

- [ ] I highly recommend you implement the design background image decorations according to the Figma design template required. 😄 

![Screenshot (308)](https://github.com/AndresEspi/portafolio/assets/78104544/b87b9eab-494f-4474-aabb-9638773c71e5)